index.bs

```diff
@@ -3011,19 +3012,21 @@ The <dfn method for=IDBObjectStore>delete(|query|)</dfn> method steps are:
 
 1. Let |range| be the result of running
     [=convert a value to a key range=] with |query| and
-    |null disallowed flag| true. Rethrow any exceptions.
+    <var ignore>null disallowed flag</var> true. Rethrow any exceptions.
 
 1. Return the result (an {{IDBRequest}}) of running
     [=asynchronously execute a request=]
-    with [=/this=] as |source| and
+    with [=/this=] as <var ignore>source</var> and
     [=delete records from an object store=] as
-    |operation|, using |store| and |range|.
+    <var ignore>operation</var>, using |store| and |range|.
```

Comment:
How I would phrase this (to avoid the "named parameter passing") is "asynchronously execute a request with this and an algorithm that performs delete records from an object store using store and range". I guess you could even factor it into two steps:

1. Let _operation_ be an algorithm that performs delete records from an object store using _store_ and _range_.
1. Asynchronously execute a request with this and _operation_.

Reply:
Good idea; this has always been an awkward part of the spec since it went algorithmic.